exercises/hamming/RUNNING_TESTS.md

```diff
@@ -0,0 +1,56 @@
+# Running Tests
+
+In order to complete this exercise, and all of the subsequent exercises, you
+will need to pass multiple tests.
+
+## Understanding Test Results
+
+After you have created and saved `hamming.rb`, run the test suite. You should
+see output like the following:
+
+    # Running:
+
+    SSSSSSESSSSSSSSS
+
+    Finished in 0.002593s, 6170.4588 runs/s, 0.0000 assertions/s.
+
+      1) Error:
+    HammingTest#test_identical_strands:
+    NameError: uninitialized constant HammingTest::Hamming
+    Did you mean?  HammingTest
+        ../hamming/hamming_test.rb:9:in `test_identical_strands'
+
+    16 runs, 0 assertions, 0 failures, 1 errors, 15 skips
+
+    You have skipped tests. Run with --verbose for details.
+
+
+The letters `SSSSSSESSSSSSSSS` show that there are sixteen tests altogether,
+that one of them has an error (`E`), and that the rest of them are skipped (all
+the `S` letters).
+
+The tests are run in randomized order, which will cause the letters to display
```

Comment:
`The tests run in randomized order` is probably preferred.  The "most correct" was in reference to the list given by @insti.

It could still be changed, but of course "need" is much too strong a word here.